### PR TITLE
Edit review of the node components section

### PR DESCRIPTION
**Pull request by radhikapc**
1. Re-ordered the sections alphabetically.
2. Edited for clarity and consistency.
3. Added consistent punctuation.

### CODE REVIEW

File: release-1.8/release_notes_draft.md [hunk 1]
```diff
@@ -422,63 +422,63 @@ kind.
...
+* [stable] Upgrade cAdvisor to v0.27.1 with the enhancement for node monitoring. [@dashpole]
 
-* metrics-server is now the reccomended way to provide the resource
-  metrics API. It is deployable as an addon, similarly to how Heapster is
-  deployed.
+  * Fix journalctl leak
+  * Fix container memory rss
+  * Fix incorrect CPU usage with 4.7 kernel
+  * OOM parser uses kmsg
+  * Add hugepages support
```
Comment (Bradamant3) [inline release-1.8/release_notes_draft.md]: hugepages appears here and under ####kubelet. do we need it in both places? (seems like not, but also seems like we'd need to ask sig-node about it)

File: release-1.8/release_notes_draft.md [hunk 2]
```diff
@@ -422,63 +422,63 @@ kind.
...
+  * Add CRI-O support
 
-##### Cluster Autoscaler
+* Sharing a PID namespace between containers in a pod is disabled by default in version 1.8. To enable for a node, use the `--docker-disable-shared-pid=false` kubelet flag. Be aware that PID namespace sharing requires docker version greater than or equal to 1.13.1.
```
Comment (Bradamant3) [inline release-1.8/release_notes_draft.md]: s/docker/Docker

File: release-1.8/release_notes_draft.md [hunk 3]
```diff
@@ -422,63 +422,63 @@ kind.
...
 
-* Cluster autoscaler is now GA
-* Incresed cluster support size to 1000 nodes
-* Respect graceful pod termination of up to 10 minutes
-* Handle zone stock-outs and failures
-* Improved monitoring and error reporting
+* Implement StatsProvider interface using CRI stats. [@yguo0905]
+
```
Comment (Bradamant3) [inline release-1.8/release_notes_draft.md]: "Implemented" to match past tense of other listitems?

File: release-1.8/release_notes_draft.md (hunk 1 above)
Comment (radhikapc) [reply release-1.8/release_notes_draft.md]: One is under the bullet point for monitoring

**Pull request merged**